### ISSUE TITLE
Amorphous head

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9839]
**Preferred term label:**
Amorphous head

**Synonyms**

**Definition (free text, please give PubMed ID)**
For illustration see: WHO laboratory manual for the examination and processing of human semen (6th Edition)

**Parent term (use hpo.jax.org/app)**
HP:0012865

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399808

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
Please provide English language definition.
Everything has a "form" (morphe) so this label does not really make much sense --

**Comment by MagoWyr** [comment]
Definition:
> 96 % of spermatozoa show irregularly shaped heads with varying alterations.

**Comment by pnrobinson** [comment]
this was discussed and dealt with previously.- > Closing
